packages/material-ui/src/ListItem/ListItem.js

```diff
@@ -22,7 +22,7 @@ export const styles = (theme) => ({
     textAlign: 'left',
     paddingTop: 8,
     paddingBottom: 8,
-    '&$focusVisible': {
+    '&$focusVisible, &:focus': {
```

Comment:
Why is the focus selector needed? If it is required for some reason we should at least isolate it to the List inside of a Menu to limit changes to other components.

Reply:
We need this so that when we ArrowLeft from a nested Menu to a parent Menu we have a visual indication of focus. It could be moved to something more specific to this work, I suppose.

Reply:
As we discussed I think this should be removed. The focusVisible style should only be visible when using the keyboard. There is an issue where using the left arrow key to return to the parent menu does not apply the focusVisible style but I think that should be handled outside of this style.